src/code/z_eff_spark.c

```diff
@@ -223,6 +227,10 @@ void EffectSpark_Draw(void* thisx, GraphicsContext* gfxCtx) {
             vertices[j].v.tc[1] = 1024;
             vertices[j].v.flag = 0;
 
+#if PLATFORM_N64
+            if (0) {}
+#endif
+
```

Comment:
can remove both these empty ifs with

```
        vertices[j].v.ob[0] = -32;
        vertices[j].v.ob[1] = -32;
        ...
        j++;
```
for all four vertices instead of j += 4